sn_cli/src/files/chunk_manager.rs

```diff
@@ -221,26 +219,26 @@ impl ChunkManager {
                     file_chunks_dir
                 };
 
-                match FilesApi::chunk_file(path, &file_chunks_dir, include_data_maps) {
-                    Ok((head_chunk_address, data_map, size, chunks)) => {
-                        progress_bar.clone().inc(1);
-                        debug!("Chunked {original_file_name:?} with {path_xor:?} into file's XorName: {head_chunk_address:?} of size {size}, and chunks len: {}", chunks.len());
-
-                        let chunked_file = ChunkedFile {
-                            head_chunk_address,
-                            file_path: path.to_owned(),
-                            file_name: original_file_name.clone(),
-                            chunks: chunks.into_iter().collect(),
-                            data_map
-                        };
-                        Ok((path_xor.clone(), chunked_file))
-                    }
-                    Err(err) => {
-                        println!("Failed to chunk file {path:?}/{path_xor:?} with err: {err:?}");
-                        error!("Failed to chunk file {path:?}/{path_xor:?} with err: {err:?}");
-                        Err(eyre!("Failed to chunk file {path:?}/{path_xor:?} with err: {err:?}"))
-                    }
-                }
+                let chop = match include_data_maps {
```

Comment:
This should be the only real change in logic. Could this really break the tests?

Reply:
Yep. Forgot to assign to self in the builder. Woops.